docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/logo/logo.scss

```diff
@@ -3,9 +3,55 @@
 //
 
 .civictheme-logo {
+  $root: &;
+
   line-height: 0;
 
   img {
     width: 100%;
   }
+
+  #{$root}--hr {
+    margin: 0.5rem;
+  }
+
+  &#{$root}-type-inline {
```

Comment:
could you please invert this so that the theme selector is a topmost rule - this is to keep this in-line with other components